### PR TITLE
Add support for Python 3.9.

### PR DESCRIPTION
**Pull request by icemac**
Plus some minor clean up in `tox.ini`.

### CODE REVIEW

**Comment by stevepiercy** [comment]
This also needs to either update [.travis.yml](https://github.com/Pylons/webtest/blob/master/.travis.yml) or switch to GitHub Actions. We switched to GHA for Pyramid, Deform, and Waitress because of the new restrictions on Travis. You can check them out for examples in `.github/workflows/`.

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ matrix:
       dist: xenial
     - python: 3.8
       env: TOXENV=py38
+    - python: 3.9
+      env: TOXENV=py39
```
Comment (icemac) [inline .travis.yml:13]: @stevepiercy `.travis.yml` is updated here.

**Comment by icemac** [comment]
But switching away from Travis-CI seems to be necessary as https://travis-ci.com/github/Pylons/webtest says: "Builds have been temporarily disabled for public repositories due to a negative credit balance."

So someone has to pay for it or to beg the Travis support for more credits.

**Comment by stevepiercy** [comment]
OK, thanks. Nonetheless Travis did not fire. This could be related to the travis-ci.org -> travis-ci.com migration.

**Comment by stevepiercy** [comment]
Ah, yes, so it is. Yeah, it's pretty much the same issue as I described in https://community.plone.org/t/travis-ci-org-shuts-down-dec-31st/13055

**Comment by icemac** [comment]
See https://github.com/zopefoundation/meta/issues/38 for the discussing of zopefoundation regarding this issue.

**Comment by stevepiercy** [comment]
I have submitted a request for more credits. We obviously cannot continue begging for more credits indefinitely.

**Comment by stevepiercy** [comment]
FYI, when migrating to GHA, the first pull request will not fire because there will be no workflow file. After one is created to trigger builds on pull requests, then it will build according to the file in the pull request. Chicken-and-egg scenario.

**Comment by stevepiercy** [comment]
I'm working on a PR to switch to GHA, and I uncovered some technical debt. See #231. To bypass the chicken-and-egg thing, I just pushed a new branch to this repo and submitted the PR against master, while ensuring that pushes to either `master` or `github-actions` will execute GHA.

**Comment by stevepiercy** [comment]
Closing as this was superceded by #231

**Pull request closed without merging**